types/storybook__polymer/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for @storybook/polymer 5.1
+// Project: https://github.com/storybookjs/storybook, https://github.com/storybookjs/storybook/tree/master/app/polymer
+// Definitions by: Joscha Feth <https://github.com/joscha>
+//                 Anton Izmailov <https://github.com/wapgear>
+//                 Dan Dean <https://github.com/dandean>
+//                 Ryan Coleman <https://github.com/ryuhhnn>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
```

Comment:
At a glance I don't think there's anything in here that requires TypeScript 3.4? This number should be the lowest possible version of TypeScript that is compatible with these definitions.